### PR TITLE
test: update isbsvc tests for getting current status

### PR DESCRIPTION
**Pull request by dpadhiar**
Explain what this PR does.

Update test cases for previous PR #1199

### CODE REVIEW

File: test/api-e2e/api_test.go [hunk 1]
```diff
@@ -57,10 +61,12 @@ func (s *APISuite) TestISBSVC() {
 	assert.Contains(s.T(), listISBSVCBody, testISBSVCName)
 
 	getISBSVCBody := HTTPExpect(s.T(), "https://localhost:8443").GET(fmt.Sprintf("/api/v1/namespaces/%s/isb-services/%s", Namespace, testISBSVCName)).
+		WithMaxRetries(5).
```
Comment (whynowy) [inline test/api-e2e/api_test.go:64]: This retry and timeout will not work. They are for single http request, what we need retry if the result is not expected.

File: test/api-e2e/api_test.go [hunk 2]
```diff
@@ -59,8 +61,21 @@ func (s *APISuite) TestISBSVC() {
...
+		default:
+			time.Sleep(100 * time.Millisecond)
+			getISBSVCBody = HTTPExpect(s.T(), "https://localhost:8443").GET(fmt.Sprintf("/api/v1/namespaces/%s/isb-services/%s", Namespace, testISBSVCName)).
+				Expect().
+				Status(200).Body().Raw()
+		}
+	}
 	assert.Contains(s.T(), getISBSVCBody, fmt.Sprintf(`"name":"%s"`, testISBSVCName))
-	assert.Contains(s.T(), getISBSVCBody, `"status":"inactive"`)
+	assert.Contains(s.T(), getISBSVCBody, `"status":"healthy"`)
```
Comment (whynowy) [inline test/api-e2e/api_test.go:78]: Do we still need this line?

File: test/api-e2e/api_test.go [hunk 3]
```diff
@@ -35,6 +35,8 @@ func (s *APISuite) TestGetSysInfo() {
 }
 
 func (s *APISuite) TestISBSVC() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```
Comment (whynowy) [inline test/api-e2e/api_test.go:38]: Should it be moved to the place before the get testing?

File: test/api-e2e/api_test.go (hunk 2 above)
Comment (dpadhiar) [reply test/api-e2e/api_test.go:78]: Not anymore since we are waiting for ISBSvc to be healthy already, removing.

**Review by whynowy — APPROVED** [review]

**Pull request merged**